react/src/overallRatings.jsx

```diff
@@ -4,11 +4,30 @@ const OverallRatings = (props) => {
+            </span>{restaurant.restaurantName} Ratings and Reviews 
+            <div className='overallrating'>
+              <span className='graph'>
+                <ul className='bargraph'>
+                  5<li class="reddeep"></li>
```

Comment:
should these be "className" instead of "class"?

Reply:
thanks Mark, it should be className. I made the adjustments